integration_tests/e2e/induction/createChangeInductionQuestionSet.cy.ts

```diff
@@ -137,6 +150,11 @@ context(`Change new Induction question set by updating 'Hoping to work on releas
               "@.previousQualifications.qualifications[1].level == 'LEVEL_4' && " +
               '@.previousTraining.trainingTypes.size() == 1 && ' +
               "@.previousTraining.trainingTypes[0] == 'HGV_LICENCE' && " +
+              '@.previousWorkExperiences.hasWorkedBefore == true && ' +
```

Comment:
Assert that the expected values were sent to in the API call to create the induction